src/js/components/DataTable/Cell.js

```diff
@@ -21,30 +19,16 @@ export const Cell = ({
 
   if (typeof content === 'string' || typeof content === 'number') {
     if (primary) {
-      content = <strong>{content}</strong>;
+      content = <Text><strong>{content}</strong></Text>;
```

Comment:
should the strong here move removed and replaced with a theme entry for weight?

Reply:
Agreed. Updated.